.travis.yml

```diff
@@ -0,0 +1,69 @@
+language: android
+jdk: oraclejdk8
+
+# Set environment variables
+env:
+    global:
+        # Set timeout for the Android emulator to come online
+        - ADB_INSTALL_TIMEOUT=10
+        # Set the minimum Android API to support
+        # Note: Not all of them work well with Travis
+        - ANDROID_TARGET=android-22
+
+before_cache:
+    - rm -f $HOME/.gradle/caches/modules-2/modules-2.lock
+
+cache:
+  directories:
+    # Cache all dirs under .gradle folder
+    - $HOME/.gradle/daemon       # Cache daemon logs
+    - $HOME/.gradle/native       # Cache library downloaded from the gradle dependency
+    - $HOME/.gradle/wrapper      # Cache the gradle
+    # Try to reuse previously downloaded Arduino resources
+    - $HOME/arduino_ide          # Cache Arduino IDE
+    - $HOME/.arduino15/packages/ # Cache Arduino IDE packages
+
+android:
+  components:
+    - tools # to get the new `repository-11.xml`
+    - tools # see https://github.com/travis-ci/travis-ci/issues/6040#issuecomment-219367943)
+    - platform-tools
+    - build-tools-28.0.3
+    - $ANDROID_TARGET # Our minimum Android version
+    - sys-img-armeabi-v7a-$ANDROID_TARGET
+    - extra-android-m2repository
+    - extra-google-m2repository
+
+before_script:
+    # Launch Android emulator to run the instrumented tests
+    - android list targets
+    - echo no | android create avd --force -n test -t $ANDROID_TARGET --abi armeabi-v7a # Create AVD for given api
+    - emulator -avd test -no-audio -no-window & # Start emulator
+    - android-wait-for-emulator
+    - adb devices # Display list of devices
+    - adb shell input keyevent 82 &
+    # Set `gradlew` as executable
+    - chmod +x $TRAVIS_BUILD_DIR/android/gradlew
+
+
+before_install:
+    # Install Android 9 SDK and accept its user agreement
+    - echo yes | sdkmanager "platforms;android-28"
+    # Install Travis CI configuration for Arduino sketches
+    - source <(curl -SLs https://raw.githubusercontent.com/adafruit/travis-ci-arduino/master/install.sh)
+
+install:
+    # Install Smartcar shield library (used by our sketch)
+    - arduino --install-library "Smartcar shield"
+
+script:
+    # Build the Android app
+    - cd $TRAVIS_BUILD_DIR/android
```

Comment:
I don't see a directory called `android` in your repository, so this step will definitely fail. What happens if you replace it with `carMobileApp` instead? :+1: